crates/rpc-types-trace/src/geth/mod.rs

```diff
@@ -123,6 +128,64 @@ pub enum GethTrace {
     JS(serde_json::Value),
 }
 
+impl GethTrace {
+    /// Try to convert the inner tracer to [DefaultFrame]
+    pub fn try_into_default(self) -> Result<DefaultFrame, UnexpectedTracerError> {
+        match self {
+            Self::Default(inner) => Ok(inner),
+            _ => Err(UnexpectedTracerError(self)),
+        }
```

Comment:
these are now redundant duplicates and can just reuse the tryfrom impl `self.try_into()`

Reply:
I removed them fully. See next comment for reasoning.